Crane/README.md

```diff
@@ -32,6 +32,37 @@ implemented using a different Activity will be displayed. In there, you can see
 embedded in Compose and Compose buttons updating the Android View. Notice how you can also
 interact with the `MapView` seamlessly.
 
+## Hilt
+
+Crane uses [Hilt][hilt] to manage its dependencies. The Hilt's ViewModel extension (with the
+`@ViewModelInject` annotation) works perfectly with Compose's ViewModel integration (`viewModel()`
+composable function) as you can see in the following snippet of code. `viewModel()` will
+automatically use the factory that Hilt creates for the ViewModel:
+
+```
+class MainViewModel @ViewModelInject constructor(
+    private val destinationsRepository: DestinationsRepository,
+    @DefaultDispatcher private val defaultDispatcher: CoroutineDispatcher,
+    datesRepository: DatesRepository
+) : ViewModel() { ... }
+
+@Composable
+fun CraneHomeContent(...) {
+    val viewModel: MainViewModel = viewModel()
+    ...
+}
+```
+
+Disclaimer: Passing dependencies that are not available at compile time to a ViewModel
+(as it's the case of the [DetailsViewModel](detailsViewModel) that takes as a dependency the name of
```

Comment:
I think this paragraph could be shorter and simplified. It's a bit wordy and long. Let me know if you'd like help re-writing.